app/views/IncomeSourceSummary.scala.html

```diff
@@ -75,13 +75,13 @@
         <p  aria-hidden="true">
             @Link.toInternalPage(
                 id=Some("updatePension"),
-                url=ApplicationConfig.incomeFromEmploymentPensionLinkUrl,
+                url=controllers.pensions.routes.UpdatePensionProviderController.doYouGetThisPension(model.empId).url,
                 value=Some(messages("tai.pension.income.details.updateLinkText"))).toHtml
         </p>
         <p class="visuallyhidden">
         @Link.toInternalPage(
             id=Some("updatePensionScreenReader"),
-            url=ApplicationConfig.incomeFromEmploymentPensionLinkUrl,
+            url=controllers.pensions.routes.UpdatePensionProviderController.doYouGetThisPension(model.empId).url,
             value=Some(messages("tai.pension.income.details.updateLinkText.screenReader"))).toHtml
```

Comment:
Not part of this change, but notice we're missing pension name on the value attrib...
tai.pension.income.details.updateLinkText.screenReader=Update or remove the pension {0}